### DISCUSSION TITLE
Prisma  Error: Cannot find module '../.schema'

### DISCUSSION THREAD

**Discussion by bobwatcherx** [TypedProject/ts-express-decorators#1844] [Q&A] [answered]
I'm following the tutorial in the TS ed documentation about prisms

and when i use it in controller i get **error not found ./schema**

```
import {BodyParams,Controller,Inject,Get,Post} from "@tsed/common";
import {UserModel, UsersRepository} from "@tsed/prisma";

@Controller("/hello-world")
export class HelloWorldController {
  @Inject()
 protected service: UsersRepository;

  @Get("/")
  get() {
    return this.service.findMany()
  }
  @Post("/add")
  add(@BodyParams() user:UserModel):Promise<UserModel>{
      return this.service.create({data:user})
  }
}
```

**Comment by Romakita** [comment, marked as answer]
Ha you generated the code with the prisma client?

**Reply by bobwatcherx** [reply]
npx prisma migrate dev
npx prisma generate

i edit helloworldcontroller.ts
my code here : https://github.com/bobwatcherx/tesedpro.git

i get error message "server bootstrap error'

**Reply by Romakita** [reply]
It's not recommanded to generate code in the src directory... because you commit the generated files. You have to ignore it in the .gitignore to avoid any potential change from one of your team dev member... (or from you). Any change on these files will be lost if you generate again the code.

Btw, this is not the problem. The custom output doesn't work apparently. I probably need to fix that.

**Reply by bobwatcherx** [reply]
> It's not recommanded to generate code in the src directory... because you commit the generated files. You have to ignore it in the .gitignore to avoid any potential change from one of your team dev member... (or from you). Any change on these files will be lost if you generate again the code.
> 
> Btw, this is not the problem. The custom output doesn't work apparently. I probably need to fix that.

there seems to be an error
in the index.js file file

node_modules/@tsed/prisma/lib/cjs

the contents of the script:
const tsed = require("../.schema");

the .schema folder is not there but is inline with index.js
so that an error appears
Error: Cannot find module '../.schema'

my code :https://github.com/bobwatcherx/tesedpro.git

**Reply by Romakita** [reply]
The fix is released ;)

**Reply by Romakita** [reply]
Hello @bobwatcherx 
Can you mark this question as answered. In future, it would be preferable to open issue for this specific usecase ;)

See you

**Comment by bobwatcherx** [comment]
I also generated code in the prisma directory but it's the same

Pada tanggal Kam, 7 Apr 2022 12.21, Romain Lenzotti <
***@***.***> menulis:
